service/host.go

```diff
@@ -400,7 +413,11 @@ func (s *host) transitionState(to State) {
 
 	// TODO(ai) this isn't used yet
 	if to < s.state {
-		s.Logger().Fatal("Can't down from state", "from", s.state, "to", to, "service", s.Name())
+		s.Logger().Fatal("Can't down from state",
```

Comment:
Again, same question about periods. Seems like maybe we should just leave them out and back out the changes towards the top to remain consistent